### PR TITLE
Update New-ADOrganizationalUnit.md

### PR DESCRIPTION
**Pull request by baardhermansen**
# PR Summary

Added a short note about accidental protection.
Changed two notes to follow Microsoft alert style.
Fixed a cursive typo. 
Minor changes to follow markdown standards.

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

File: docset/winserver2022-ps/activedirectory/New-ADOrganizationalUnit.md [hunk 1]
```diff
@@ -24,12 +25,13 @@ New-ADOrganizationalUnit [-WhatIf] [-Confirm] [-AuthType <ADAuthType>] [-City <S
...
 ## DESCRIPTION
+
 The **New-ADOrganizationalUnit** cmdlet creates an Active Directory organizational unit (OU).
 You can set commonly used OU property values by using the cmdlet parameters.
-**Property** values that are not associated with cmdlet parameters can be set by using the* OtherAttributes* parameter.
+**Property** values that are not associated with cmdlet parameters can be set by using the *OtherAttributes* parameter.
 
 You must set the *Name* parameter to create a new OU.
-If you do not specify the *Path* parameter, the cmdlet creates an OU under the default NC head for the domain.
+If you do not specify the *Path* parameter, the cmdlet creates an OU under the default NC (Naming Context) head for the domain.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/activedirectory/New-ADOrganizationalUnit.md:34]:
```suggestion
If you do not specify the *Path* parameter, the cmdlet creates an OU under the default naming context (NC) head for the domain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @baardhermansen, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
Can you please check the other OS Versions of the article? Thank you

**Comment by baardhermansen** [comment]
> Can you please check the other OS Versions of the article? Thank you

PRs opened :)

(I didn't even realize there were separate documents! For some reason i thought there was one master document and only sections within it that was updated according to the OS versions.)

**Comment by get-itips** [comment]
Thank you, you could also edit them in this same Pull Request

**Comment by baardhermansen** [comment]
> Thank you, you could also edit them in this same Pull Request

Yeah, i realized that just a bit too late...

**Comment by baardhermansen** [comment]
@get-itips 
> Thank you, you could also edit them in this same Pull Request

FYI: I've closed the other PRs and incorporated all changes into this one PR.

**Comment by get-itips** [comment]
Thanks

**Pull request merged**